### PR TITLE
Remove management of ol.source.ImageVector

### PR DESCRIPTION
**Pull request by oterral**
This PR prepare for v4.6.0 of openlayer removing use of ol.source.ImageVector.

### CODE REVIEW

**Comment by gberaudo** [comment]
Thanks @oterral.
I have to release a new version of OL-Cesium before I can merge your PR.

**Comment by oterral** [comment]
Ok

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "googshift": "0.9.2",
     "jsdoc": "3.5.5",
     "nomnom": "1.8.1",
-    "openlayers": "4.5.0",
```
Comment (fredj) [inline package.json]: please use version 4.6.0

Comment (oterral) [reply package.json]: Done

**Comment by gberaudo** [comment]
@oterral, there is now a 4.6.1, can you update once more and add a line in CHANGES.md?

**Comment by oterral** [comment]
Done.

**Comment by gberaudo** [comment]
Thanks

**Pull request merged**